diff --git a/dist/app/plugins/payload.client.js b/dist/app/plugins/payload.client.js
index 98e0ece07aac4cb2f98955abaf2afb40456f7551..51cd584920f1ca84b1b5ead7db1f2d1b77296444 100644
--- a/dist/app/plugins/payload.client.js
+++ b/dist/app/plugins/payload.client.js
@@ -46,7 +46,7 @@ const plugin = /* @__PURE__ */ defineNuxtPlugin({
 						const { rel: _rel, ...rest } = link;
 						return {
 							...rest,
-							rel: "prefetch"
+							rel: link.as === "image" ? _rel : "prefetch"
 						};
 					}) });
 					forwardedPrefetchEntries.set(pathname, entry);
